### PR TITLE
Ingestion: EAMT 2024

### PR DESCRIPTION
**Pull request by anthology-assist**
1. [x] In the Github sidebar, add the PR to work items and the current milestone
1. [x] In the Github sidebar, under "Development", make sure to link to the corresponding issue
1. [x] Make sure the branch is merged with the latest `master` branch
1. [x] Ensure that there are editors listed in the `<meta>` block
1. [x] For workshops, add a `<venue>ws</venue>` tag to its meta block
1. [x] For workshops, add a backlink from the main event's `<event>` block
1. [ ] Add events to their relevant SIGs
1. [x] Look at the venue listing for prior years, and ensure that the new volume titles are consistent. You can do this by clicking on the venue name from a paper page, which will take you to the vendor listing.
1. [x] Navigate to the event page preview (e.g., https://preview.aclanthology.org/icnlsp-ingestion/events/icnlsp-2021/), and page through, to see if there are any glaring mistakes
1. [x] Skim through the complete listing, looking for mis-parsed author names.
1. [x] Download the frontmatter and verify that the table of contents matches at least three randomly-selected papers
1. [x] Download 3–5 PDFs (including the first and last one) and make sure they are correct (title, authors, page numbers).

### CODE REVIEW

File: data/xml/2024.gitt.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      <editor><first>Luisa</first><last>Bentivogli</last></editor>
+      <editor><first>Joke</first><last>Daems</last></editor>
+      <editor><first>Eva</first><last>Vanmassenhove</last></editor>
+      <editor><first>Jasmijn</first><last>Bastings</last></editor>
+      <publisher>European Association for Machine Translation (EAMT)</publisher>
+      <address>Sheffield, United Kingdom</address>
+      <month>June</month>
+      <year>2024</year>
+      <url hash="3b54253c">2024.gitt-1</url>
+      <venue>gitt</venue>
```
Comment (mjpost) [inline data/xml/2024.gitt.xml:17]: The gitt.yaml file has "First workshop" in the venue title; should be changed to just "workshop".

image

**Review by mjpost — COMMENTED** [review]
LGTM except for the small comment

**Comment by davidstap** [comment]
@anthology-assist it seems like EAMT Main Volume 1 and Volume 2 had errors and have been updated a few hours ago, see #3281:

> Dear @davidstap We just noticed some errors in the previous. Could you help us update main vol1 and 2 please?
> 
> EAMT Main Volume 1: Research And Implementations & Case Studies: https://drive.google.com/drive/u/2/folders/1DFizXslJ1QjArkYShR7aiLjrvTdDlMlv
> 
> EAMT Main Volume 2: Products & Projects: https://drive.google.com/drive/u/2/folders/1hDUtay-Yzt3cVzMnK4GwgSjnZwVP_qw5



> Dear @davidstap We just noticed some errors in the previous. Could you help us update main vol1 and 2 please?
> 
> EAMT Main Volume 1: Research And Implementations & Case Studies: https://drive.google.com/drive/u/2/folders/1DFizXslJ1QjArkYShR7aiLjrvTdDlMlv
> 
> EAMT Main Volume 2: Products & Projects: https://drive.google.com/drive/u/2/folders/1hDUtay-Yzt3cVzMnK4GwgSjnZwVP_qw5

File: data/xml/2024.gitt.xml (hunk 1 above)
Comment (anthology-assist) [reply data/xml/2024.gitt.xml:17]: Addressed

**Comment by anthology-assist** [comment]
Reingested.

**Review by mjpost — APPROVED** [review]

**Pull request merged**